docs/src/main/asciidoc/kafka-dev-services.adoc

```diff
@@ -0,0 +1,40 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Dev Services for Kafka
```

Comment:
Does it really make sense to have a specific document just for that?

Not sure how it's organized for others but it feels a bit weird.

Reply:
I don't want to have it in the Kafka quickstart (which has a lot of extra content that needs to move). Dev Services and its configuration has nothing to do with the quickstart. When we will have a proper reference guide it will go there. 

Having a Kafka reference guide is on my todo list.